apps/kcapi-hasher.c

```diff
@@ -1125,6 +1130,9 @@ int main(int argc, char *argv[])
 				version(argv[0]);
 				ret = 0;
 				goto out;
+			case 'T':
+				targetfile = optarg;
```

Comment:
Should this use a separate variable that is ignored if -c is also not passed in?

And have targetfile set later if both -c and -T were used?

Reply:
That isnt necessary as `targetfile` will never be used if `-c` is specified. But I should add a check whether -T is used in fipscheck and error out.